aerleon/lib/packetfilter.py

```diff
@@ -282,22 +291,22 @@ def _CheckAddressAf(self, addrs):
 
     def _FormatPart(
         self,
-        action,
-        direction,
-        log,
-        interface,
-        af,
-        proto,
-        src_addr,
-        src_port,
-        dst_addr,
-        dst_port,
-        tcp_flags_set,
-        tcp_flags_check,
-        icmp_types,
-        options,
-        stateful,
-    ):
+        action: str,
+        direction: str,
+        log: List[VarType],
+        interface: Optional[str],
+        af: str,
+        proto: List[str],
+        src_addr: str,
+        src_port: Union[List[Any], str],
```

Comment:
Can we constrain the Any here? Is this just a list of nacaddr objects?

Reply:
List of strings actually since these are ports. Nacaddr are for addresses only.